python/pyfunc-server/pyfuncserver/protocol/rest/handler.py

```diff
@@ -57,12 +57,14 @@ def post(self, full_name: str):
         request = self.validate(self.request)
         headers = self.get_headers(self.request)
 
-        response = model.predict(request, headers=headers)
-
-        response_json = orjson.dumps(response)
+        output = model.predict(request, headers=headers)
+        response_json = orjson.dumps(output.http_response)
         self.write(response_json)
         self.set_header("Content-Type", "application/json; charset=UTF-8")
 
+        if self.publisher is not None and output.contains_prediction_log:
```

Comment:
Do we not need to invoke this method as `output.contains_prediction_log()` ? Otherwise it will always evaluate to `True`.

Reply:
🤦 I forgot to add `()`, will add that